backend/tools/setup_cgov_env.sh

```diff
@@ -31,6 +31,8 @@ function setup_cgov_env {
     # Logging level, (critical, error, warning, info and debug). Default to info
     export NEW_RELIC_LOG_LEVEL=info
 
+    export NEW_RELIC_CONFIG_FILE=newrelic.ini
```

Comment:
```suggestion
    # Used for configuring the logging details sent to new relic
    export NEW_RELIC_CONFIG_FILE=newrelic.ini
```

or something similar, just so we know in the future adjustments to tuning NR for our needs goes in here, so long as it doesn't contain sensitive data.